server/utilities/send-sms.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config()
+
+const accountSid = process.env.TWILIO_ACCOUNT_SID;
+const authToken = process.env.TWILIO_AUTH_TOKEN;
+const client = require('twilio')(accountSid, authToken);
+
+const sendTwilioSMS = async (customerPhone, message) => {
+  try {
+    const sentMessage = await client.messages.create({
+      body: message,
+      from: '+12282830492',
```

Comment:
Do you see this as a potential issue in the future?
No need to change it, but I want you to think about this.
Hardcoding this value can cause some issues in the future. Each twilio number has a limit of text messages that can be sent per day. 
Ideally if this request fails based on the quota rate, you would want to retry to send the text message with a new phone number that twilio allocates you. However, we only have a free plan and they only give you one number.